src/pages/Admin/Charity/Permissions/Form.tsx

```diff
@@ -0,0 +1,47 @@
+import { ErrorMessage } from "@hookform/error-message";
+import { getTypedKeys } from "helpers";
+import MobileTable from "./MobileTable";
+import Table from "./Table";
+import useSubmit from "./useSubmit";
+
+export default function Form() {
+  const { disabled, errors, reset, submit } = useSubmit();
+
+  const errorName = getTypedKeys(errors).find((errKey) => !!errors[errKey]);
+
+  return (
+    <form
+      className="grid gap-6 w-full"
+      onSubmit={submit}
+      onReset={() => reset()}
+    >
+      <Table className="max-lg:hidden" disabled={disabled} />
```

Comment:
Used webapp width for breakpoints instead of tailwind container queries as there seem to be some bugs in it, see example below.

Notice a couple of things:
- using named @container like `@container/perm`
- `Table` should be displayed only when `@container/perm` min width is `md:`
- `MobileTable` should be hidden only when `@container/perm` min width is `md:`
- component with `@container/perm` class name is clearly less than `md:` in width
- `Table` is still appearing
![image](https://user-images.githubusercontent.com/19427053/225577828-03f16251-f978-448c-bedf-f74800769607.png)